documentation/tut/toy-example-process.rst

```diff
@@ -0,0 +1,124 @@
+        The sensor recording the power of the SHIFTABLE load is Power (SHIFTABLE) (ID: 6).
+
+
+
+Trigger an updated schedule
+----------------------------
+
+In this example, we are planning to consume at a 200kW constant power for a period of 4h. 
+
+This load is to be schedule for tomorrow, except from the period from 3pm to 4pm (imposed using the ``--forbid`` flag).
+
+
+Now we are ready to schedule a process. Let's start with the INFLEXIBLE policy, the simplest.
+
+.. code-block:: bash
+
+    flexmeasures add schedule for-process --sensor-id 4 --consumption-price-sensor 1\
+      --start ${TOMORROW}T00:00:00+02:00 --duration PT24H --process-duration PT4H \
+      --process-power 0.2MW --process-type INFLEXIBLE \ 
+      --forbid "{\"start\" : \"${TOMORROW}T15:00:00+02:00\", \"duration\" : \"PT1H\"}"
+
+Under the INFLEXIBLE policy, the process starts as soon as possible, in this case, coinciding with the start of the planning window.
+
+Following the INFLEXIBLE policy, we'll schedule the same 4h block using a BREAKABLE policy.
+
+.. code-block:: bash
+
+    flexmeasures add schedule for-process --sensor-id 5 --consumption-price-sensor 1\
+      --start ${TOMORROW}T00:00:00+02:00 --duration PT24H --process-duration PT4H \
+      --process-power 0.2MW --process-type BREAKABLE \ 
+      --forbid "{\"start\" : \"${TOMORROW}T15:00:00+02:00\", \"duration\" : \"PT1H\"}"
+ 
+The BREAKABLE policy splits or breaks the process into blocks that can be scheduled discontinuously. The smallest possible unit is (currently) determined by the sensor's resolution. 
+
+Finally, we'll schedule the process using the SHIFTABLE policy.
+
+.. code-block:: bash
+
+    flexmeasures add schedule for-process --sensor-id 6 --consumption-price-sensor 1\
+      --start ${TOMORROW}T00:00:00+02:00 --duration PT24H --process-duration PT4H \
+      --process-power 0.2MW --process-type SHIFTABLE \ 
+      --forbid "{\"start\" : \"${TOMORROW}T15:00:00+02:00\", \"duration\" : \"PT1H\"}"
+ 
+
+Results
+---------
+
+The image below show the resulting schedules following each of the three policies. You will see similar results in your `FlexMeasures UI <http://localhost:5000/assets/4/>`_. 
+
+ 
+.. image:: https://github.com/FlexMeasures/screenshots/raw/main/tut/toy-schedule/asset-view-process.png
+    :align: center
+|
+
+
+In the first policy, there's no flexibility and it needs to schedule the process as soon as possible. 
+Meanwhile, in the BREAKABLE policy, the consumption blocks surrounds the time restriction to consume in the cheapest hours. Among the three polices, the BREAKABLE policy can achieve the best 
+Finally, in the SHIFTABLE policy, the process is shifted to capture the best prices, avoiding the time restrictions.
+
+
+Quantitatively, comparing the total cost of running the process under each policy, the BREAKABLE policy achieves the best results. This is because it can fit much more consumption blocks in the cheapest hours.
```

Comment:
I didn't understand right away what the cells contain, so I would add:

"Let's list the power price the policies achieved for each of the four blocks they scheduled:"

Reply:
Good addition, that way it gets easier :smile: